packages/xstate-react/test/useMachine.test.tsx

```diff
@@ -137,7 +137,7 @@ describe('useMachine hook', () => {
         execute: false
       });
 
-      expect(service.initialized).toBe(false);
+      // expect(service.initialized).toBe(false);
```

Comment:
I think this wasnt relevant for this particular test and it just broke by starting service immediately.